makefile

```diff
@@ -0,0 +1,4 @@
+CC = g++
+CFLAGS = -std=c++11
```

Comment:
How about choosing a cross-platform build tool like cmake so we can build in windows.